gpApi/index.js

```diff
@@ -339,12 +301,8 @@ const gpApi = {
     },
   },
   voterData: {
-    locations: {
-      url: `${base}voter-data/locations`,
-      method: 'GET',
-      withAuth: true,
-    },
     pathToVictory: {
+      // TODO: not migrated to nest yet!!!
```

Comment:
Do we have a ticket in Jira for this?  Can we annotate this comment w/ the ticket # if we do? 🙏